fvtest/tril/examples/mandelbrot/main.cpp

```diff
@@ -45,8 +49,13 @@ int main(int argc, char const * const * const argv) {
     assert(mandelbrotCompiler.compile() == 0);
     auto mandelbrot = mandelbrotCompiler.getEntryPoint<MandelbrotFunction*>();
 
+#if __cplusplus <= 199711L
+    const auto size = 80;                   // number of rows/columns in the output table
+    const auto iterations = 1000;           // number of iterations to be performed
+#else
     constexpr auto size = 80;                   // number of rows/columns in the output table
```

Comment:
Not sure why we are using constexpr here (instead of just const).

Reply:
I think I'd just keep the const auto versions.